docs/PacketFence_Installation_Guide.asciidoc

```diff
@@ -4535,6 +4535,207 @@ Now go in `Nodes` and click on a MAC address and you will see that a new tab app
+
+NOTE: This command will create configuration scripts in `/etc/monit.d/`, but it will not remove old(er) scripts from an earlier installation. During an upgrade any unused `/etc/monit.d/*.conf` files will need to be removed manually.
+
+
+Include the generated configurations in the monit config
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+RHEL / CentOS
+----
+sed -i -e 's/^include.*//g' /etc/monit.conf && echo "include /etc/monit.d/*.conf" >> /etc/monit.conf
+----
+
+Debian
+----
+sed -i -e 's/include.*\*$//g' /etc/monit/monitrc && echo "include /etc/monit/conf.d/*.conf" >> /etc/monit/monitrc
+----
+
+Remove the old monit script
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+This step is only required during an upgrade from earlier versions.
+----
+rm /etc/monit.d/packetfence.monit
+----
+
+Run the monitoring script update to fetch the scripts and config
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Ensure the script outputs: `Update completed successfully`
+
+RHEL / CentOS
+----
+yum install uuid -y
+/usr/local/pf/addons/monit/monitoring-scripts/update.sh
+----
+
+Debian
+----
+apt-get install uuid-runtime
+/usr/local/pf/addons/monit/monitoring-scripts/update.sh 
+----
+
+Ensure the pf group can write the logs
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+----
+chmod g+w /usr/local/pf/logs/*
+----
+
+Run the monitoring scripts
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+----
+/usr/local/pf/addons/monit/monitoring-scripts/run-all.sh
+----
+
+Ensure this script outputs `No error to report`
+
+Error: Syslog is not in asynchronous mode
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This step is only required if a `Syslog is not in asynchronous mode` error is received above.
+
+In `/etc/rsyslog.conf`, replace:
+----
+*.info;mail.none;authpriv.none;cron.none                /var/log/messages
+----
+
+with (a hyphen):
+----
+*.info;mail.none;authpriv.none;cron.none                -/var/log/messages
+----
+
+Restart monit
+^^^^^^^^^^^^^
+----
+systemctl restart monit
+----
+
+Installing a MTA
+^^^^^^^^^^^^^^^^
+A MTA is needed to correctly relay emails from monit. If _localhost_ is used as smtpserver, make sure that a MTA is installed and configured on the server. 
+
+RHEL / CentOS
+----
+yum install mailx -y
+----
+
+Debian
+----
+apt-get install heirloom-mailx
+----
+
+Test the MTA
+^^^^^^^^^^^^
+----
+mail -s "This is a test" user@example.com
+
+[CTRL+D]
+----
+
+
+Monit Summary
+^^^^^^^^^^^^^
+
+View the monit summary to ensure all services are status `Running`, `Accessible`, or `Status ok`. Address any services that display any other failed status. Monit will display the services in the same order that they are processed. If the summary appears stuck, troubleshoot the next service in the list.
+
+----
+monit summary
+----
+
+NOTE: `patch` updates only once a week. It is normal to see status `Waiting`.
+
+
+packetfence-etcd
```

Comment:
It doesn't look like its really the place for this here since its unrelated to monit.

This belongs more in the clustering guide

See #3451